### DISCUSSION TITLE
[Urgent]Certificate does not have a valid expiration date.

### DISCUSSION THREAD

**Discussion by SurendarG** [OxalisCommunity/oxalis#533] [Q&A]
Hi Friends,

After Peppol certificate renewal, we are getting 'Certificate does not have a valid expiration date.' error. After renewal our cert expiry date is 20 Mar 2021.

we are using Oxalies-AS4 4.1 and Java 1.8.0_45.

[Oxalies Error-20210330.txt](https://github.com/OxalisCommunity/oxalis/files/6225973/Oxalies.Error-20210330.txt)

**Comment by aaron-kumar** [comment]
You mentioned:
> "After renewal our cert expiry date is 20 Mar 2021."

Is there typo -"2021"? or Are you using expired certificate?


Attached inbound/tomcat logs does not print any error. In fact, it says that Oxalis web application deployed successfully. Please share relevant logs with error. 

Is it so that some sending AP or your standalone client facing such error? If so, then can you share standalone logs?

P.S. Please mention complete versions info of both Oxalis and Oxalis-AS4.

**Comment by aaron-kumar** [comment]
@SurendarG  Please provide relevant information, otherwise issue will be moved to discussion....
